### ISSUE TITLE
reverseString - typo in comment

### ISSUE THREAD

**Issue by philipp007** [StephenGrider/AlgoCasts#11]
There seems to be a typo in AlgoCasts/exercises/reversestring/index.js: 
//   reverse('apple') === 'leppa' 

Here the expected value should be 'elppa' instead of 'leppa'

**Comment by nathanhannig** [comment]
Why did you close this issue? I came to report same issue and since it is not fixed it should not be closed yet. Please consider re-opening; otherwise others may create a duplicate.

**Comment by philipp007** [comment]
I closed it because I saw that there is already a Pull request for this: https://github.com/StephenGrider/AlgoCasts/pull/1

**Comment by nathanhannig** [comment]
great, ideally all pull requests that fix an issue should have an open issue item and closed once the pull request is merged. I think the issue should be set to Open status still.

**Comment by philipp007** [comment]
Allright then :)

**Comment by jwarner112** [comment]
Duplicate of #1 (I know you described it already, I'm just rounding up all the duplicates for `leppa`)
